### PR TITLE
Add new colors and font for quotes

### PR DESCRIPTION
**Pull request by MelenkoOleksandr**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/styles/_variables.scss [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
 $mainFont: "Montserrat", sans-serif;
-
+$quoteFont: "Norican", cursive;
+ 
+$text: #000000; 
+$secondaryColor : #180C30;
+$purpleHeading: #581184;
+$lightPinkBackground: #F0EAF4; 
+$mainColor: #FFFFFF;
+$deleteColor: #C91212;
```
Comment (andrii-kuzmichov) [inline src/styles/_variables.scss]: If you create these variables in `_variables.scss` file, then should be use color variables for each value from `_colors.scss`, also would be better move all color variables to `_colors.scss` file

File: src/styles/_variables.scss [hunk 2]
```diff
@@ -1,15 +1,5 @@
...
-$black: #000;
-$white: #fff;
-$gray: #f2f2f2;
-$lightThemeBackground: #fff;
-$lightThemeAccept: #6c488a;
-$lightThemeHeader: #696969;
-
-$darkThemeBackground: #180c30;
+@import '_colors.scss';
+@import '_fonts.scss';
```
Comment (andrii-kuzmichov) [inline src/styles/_variables.scss]: Why you need this import here?

File: src/styles/_colors.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+$lightThemeBackground: #fff;
+$lightThemeAccept: #6c488a;
+$lightThemeHeader: #696969;
+  
+$darkThemeBackground: #180c30;
+
+$transparentBlackBackground: #00000054;
+$headerBackground: #f7f4f466;
+$mainSearchGrey: #c4c4c475;
+$transparentGreyBackground: #74747466;
```
Comment (andrii-kuzmichov) [inline src/styles/_colors.scss]: Please, should be created color theme and reuse for specific params

File: src/styles/_variables.scss (hunk 2 above)
Comment (MelenkoOleksandr) [reply src/styles/_variables.scss]: Should I import them in the main.scss: file?

Comment (andrii-kuzmichov) [reply src/styles/_variables.scss]: fonts file for _fonts.scss
and colors for each file what you need

Comment (MelenkoOleksandr) [reply src/styles/_variables.scss]: If I delete this imports site will crash

File: src/components/landing/EventCard/IconsEventCard/IconsEventCard.scss [hunk 4]
```diff
@@ -22,7 +22,7 @@ $postition-icons-list-right: 75px;
   height: 40px;
   width: 40px;
   background-color: $lightThemeAccept;
-  box-shadow: 0px 4px 4px $transparentBlackBackground;
+  box-shadow: 0px 4px 4px  $transparentBlackBackground;
```
Comment (andrii-kuzmichov) [inline src/components/landing/EventCard/IconsEventCard/IconsEventCard.scss]: remove extra space

File: src/styles/colors.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
+$black: #000000;
+$white: #FFFFFF;
+$gray: #f2f2f2;
+
+$text: $black; 
+$secondaryColor : #180C30;
+$purpleHeading: #581184;
+$lightPinkBackground: #F0EAF4; 
+$mainColor: $white;
+$deleteColor: #C91212;
```
Comment (andrii-kuzmichov) [inline src/styles/colors.scss]: Create variables for specific colors or move each to above of file near color theme

File: src/components/footer/footer.scss [hunk 6]
```diff
@@ -45,11 +48,11 @@
 .custom-footer__subscribe input {
     background-color: $mainSearchGrey;
     font-family:  $mainFont;
-font-style: normal;
-font-weight: 700;
-font-size: $fontsize-14;
-display: flex;
-color: $headerBackground;
+    font-style: normal;
+    font-weight: 700;
```
Comment (andrii-kuzmichov) [inline src/components/footer/footer.scss]: create and use variables for `font-weight: 700;`

File: src/components/Header/dropdownMenu/dropdownMenu.scss [hunk 7]
```diff
@@ -1,11 +1,13 @@
...
+@import "../../../styles/colors.scss";
+@import "../../../styles/fonts.scss";
 
 .user-name {
   margin: 0 3px 0 5px;
   display: flex;
   flex-direction: column;
   justify-content: center;
-  color: $white;
+  color: white;
```
Comment (andrii-kuzmichov) [inline src/components/Header/dropdownMenu/dropdownMenu.scss]: typo? Why you use static color?

**Review by andrii-kuzmichov — APPROVED** [review]

**Pull request merged**